src/flask_ml/flask_ml_server/models.py

```diff
@@ -1,192 +1,45 @@
-
-
-class FileResult(MLResult):
-    result: str = Field(..., description="Path of the result file.", min_length=1)
-
-
-class ImageResult(FileResult):
-    pass
-
-
-class VideoResult(FileResult):
-    pass
-
-
-class AudioResult(FileResult):
-    pass
-
-
-class TextResult(MLResult):
-    result: str = Field(..., description="The result text.", min_length=1)
-
-
-class BatchTextResult(BaseModel):
-    results: List[TextResult] = Field(
-        ..., description="List of text results", min_length=1
-    )
-
-
-class BatchImageResult(BaseModel):
-    results: List[ImageResult] = Field(
-        ..., description="List of image results", min_length=1
-    )
-
-
-class BatchAudioResult(BaseModel):
-    results: List[AudioResult] = Field(
-        ..., description="List of audio results", min_length=1
-    )
-
-
-class BatchVideoResult(BaseModel):
-    results: List[VideoResult] = Field(
-        ..., description="List of video results", min_length=1
-    )
-
-
-class BaseResponseModel(BaseModel):
-    """
-    Base model for response models.
-    Methods:
-        model_dump_json() -> str:
-            Returns the JSON representation of the model.
-    """
-
-    def get_response(self, status_code: int = 200):
-        return Response(
-            response=self.model_dump_json(),
-            status=status_code,
-            mimetype="application/json",
-        )
-
-
-class ResponseModel(BaseResponseModel):
-    """
-    Model representing the results from an ML model.
-    Attributes:
-        status (str): The status of the operation, e.g., 'success'
-        results (List[MLResult]): List of results.
-    Methods:
-        get_response(status_code: int = 200) -> Response:
-            Returns a Flask Response object with the JSON representation of the model.
-    """
-
-    status: str = Field(
-        default="SUCCESS",
-        description="The status of the operation, e.g., 'SUCCESS'",
-        min_length=1,
-    )
-    results: Union[
-        BatchTextResult, BatchVideoResult, BatchAudioResult, BatchImageResult
-    ] = Field(
-        ...,
-        description="List of results",
-    )
-
-
-class ErrorResponseModel(BaseResponseModel):
-    """
-    Model representing an error response.
-    Attributes:
-        status (str): The status of the operation, e.g., 'error'
-        detail (str): Details about the error that occurred.
-    Methods:
-        get_response(status_code: int = 400) -> Response:
-            Returns a Flask Response object with the JSON representation of the model.
-    """
-
-    status: str = Field(
-        default="ERROR",
-        description="The status of the operation, e.g., 'ERROR'",
-        min_length=1,
-    )
-    errors: List = Field(..., description="Details about the error that occurred")
-
-    def get_response(self, status_code: int = 400):
-        return super().get_response(status_code)
+from flask import Response
+from pydantic import model_validator
+from ._generated_models import *
+
```

Comment:
`models.py` module wildcard imports generated models to prevent breaking other imports. It includes the custom validators from before. However, these can be eventually put directly into the spec. For example, the data type can become a property of the FileInput, TextInput etc. In this case, the data type will always be correct, without needing any custom validation.